service/project_membership.go

```diff
@@ -9,36 +9,43 @@ import (
 	"release-manager/service/model"
 
 	"github.com/google/uuid"
+	"go.strv.io/background"
+	"go.strv.io/background/task"
 )
 
 type ProjectMembershipService struct {
 	authSvc    model.AuthService
 	projectSvc model.ProjectService
 	repository model.ProjectInvitationRepository
+	mailer     model.Mailer
```

Comment:
Here and for the rest of the services above, it feels weird that interfaces are a part of model package.

Wouldn't it be smoother to at least have it simply near `service.go`?.. 🤔

Reply:
I created a separate PR #32